templates/zerver/help/include/sidebar_index.md

```diff
@@ -52,8 +52,10 @@
 * [Configure default view](/help/configure-default-view)
 * [Enable full width display](/help/enable-full-width-display)
 * [Display the buddy list on narrow screens](/help/move-the-users-list-to-the-left-sidebar)
+* [Display organization name in left-sidebar](/help/display-organization-name)
```

Comment:
Let's use "Display organization name" for the title here, so that it's not overly long.